sed/loader/flash/utils.py

```diff
@@ -1,62 +0,0 @@
-"""Helper functions for the flash loader."""
```

Comment:
I think this is acciedently deleted

Reply:
No, I moved the content (one function) into the general utils, as it is also used in the sxp loader now.